### ISSUE TITLE
add user entry to db with /helthtoday command

### ISSUE THREAD

**Issue by tsnaik** [tsnaik/helth-bot#18]

**Comment by pradhan1234** [comment]
How can we link issues in order of their dependencies?

For example this depends on https://github.com/tsnaik/helth-bot/issues/17

**Comment by tsnaik** [comment]
I don't think we can do this, but in our project we can arrange them in the order of priority

**Comment by pradhan1234** [comment]
priority ordering sounds good

**Comment by pradhan1234** [comment]
Was looking at this, looks like it'll suit our usecase
https://github.com/marketplace/actions/blocking-issues
